config/manager/manager.yaml

```diff
@@ -25,36 +25,42 @@ spec:
       securityContext:
         runAsNonRoot: true
       containers:
-      - args:
-        - --leader-elect
-        image: controller:latest
-        name: manager
-        imagePullPolicy: Always
-        securityContext:
-          allowPrivilegeEscalation: false
-        livenessProbe:
-          httpGet:
-            path: /healthz
-            port: 8081
-          initialDelaySeconds: 15
-          periodSeconds: 20
-        readinessProbe:
-          httpGet:
-            path: /readyz
-            port: 8081
-          initialDelaySeconds: 5
-          periodSeconds: 10
-        resources:
-          limits:
-            cpu: 200m
-            memory: 100Mi
-          requests:
-            cpu: 100m
-            memory: 20Mi
-        env:
-          - name: WATCH_NAMESPACE
-            valueFrom:
-              fieldRef:
-                fieldPath: metadata.annotations['olm.targetNamespaces']
+        - args:
+            - --health-probe-bind-address=:8081
+            - --metrics-bind-address=127.0.0.1:9090
```

Comment:
Added the port binding directly to manager.yaml

Reply:
will 127.0.0.1 not make the metrics only available to the pod itself but not externally?